submissions/yhorodechna/html-css-popup/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>popup</title>
+    <link rel="stylesheet" href="css/null_style.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width , initial-scale=1.0" />
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="popup__body">
+                <input tabindex="1" class="popup__input" type="checkbox" id="popup__input" />
+                <label for="popup__input" class="popup__label  menu__item">
+                    <img src="img/header/popup-button.png" alt="popup picture" />
+                </label>
+                <nav class="popup">
+                    <ul class="popup__list">
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/calendar.png" alt="calendar picture" />
+                            <span>Calendar</span>
```

Comment:
Logically elements inside the popup are a list of links.

Reply:
Earlier I used links , but decided to deleted them . Now returned them.